<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customer_report</class>
 <widget class="QWidget" name="customer_report">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="balance">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>549</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>52</y>
     <width>690</width>
     <height>483</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="ggg_6">
    <attribute name="title">
     <string>فواتير المبيعات</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_12">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>672</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_36">
      <attribute name="title">
       <string>فواتبر نقدي</string>
      </attribute>
      <widget class="QTableView" name="puchase_cash_view_6">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>1</y>
         <width>655</width>
         <height>382</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_37">
      <attribute name="title">
       <string>فواتير اجل</string>
      </attribute>
      <widget class="QTableView" name="puchase_credit_view_6">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>1</y>
         <width>655</width>
         <height>382</height>
        </rect>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QLineEdit" name="pur_cash_total_6">
     <property name="geometry">
      <rect>
       <x>451</x>
       <y>431</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="pur_credit_total_6">
     <property name="geometry">
      <rect>
       <x>226</x>
       <y>431</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="pur_total_6">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>430</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>432</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع المبيعات</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>433</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع الفواتبر الاجل</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>561</x>
       <y>433</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع الفواتبر النقدي</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_38">
    <attribute name="title">
     <string>مرتجعات المبيعات</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_13">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>2</y>
       <width>672</width>
       <height>421</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_39">
      <attribute name="title">
       <string>فواتبر نقدي</string>
      </attribute>
      <widget class="QTableView" name="pur_cash_return_view_6">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>3</y>
         <width>654</width>
         <height>378</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_40">
      <attribute name="title">
       <string>فواتير اجل</string>
      </attribute>
      <widget class="QTableView" name="pur_credit_return_view_6">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>4</y>
         <width>654</width>
         <height>378</height>
        </rect>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>427</y>
       <width>85</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع المرتجعات</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pur_re_credit_total_6">
     <property name="geometry">
      <rect>
       <x>227</x>
       <y>426</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>562</x>
       <y>428</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع الفواتبر النقدي</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pur_re_cash_total_6">
     <property name="geometry">
      <rect>
       <x>452</x>
       <y>426</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="pur_re_total_6">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>425</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_52">
     <property name="geometry">
      <rect>
       <x>346</x>
       <y>428</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع الفواتبر الاجل</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_41">
    <attribute name="title">
     <string>دفعات</string>
    </attribute>
    <widget class="QTableView" name="payment_view_6">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>5</y>
       <width>676</width>
       <height>409</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLineEdit" name="payment_total_6">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>420</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>141</x>
       <y>422</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع الدفعات</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_42">
    <attribute name="title">
     <string>جميع العملاء</string>
    </attribute>
    <widget class="QTableView" name="supplier_view_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>3</y>
       <width>678</width>
       <height>414</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_54">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>425</y>
       <width>87</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>مجموع المديونية</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="total_balance_6">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>424</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>534</x>
       <y>424</y>
       <width>116</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>اخفاء الارصدة </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>552</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>رصيد</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>248</x>
     <y>15</y>
     <width>262</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="4,1">
    <item>
     <widget class="QComboBox" name="customer">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_55">
      <property name="text">
       <string>العميل</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>طباعة</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
